src/reducers/existingThoughtMove.js

```diff
@@ -40,7 +40,7 @@ export default (state, { oldPath, newPath, offset }) => {
   const sameContext = equalArrays(oldContext, newContext)
   const oldThought = getThought(value, thoughtIndex)
   const newThought = removeDuplicatedContext(moveThought(oldThought, oldContext, newContext, oldRank, newRank), newContext)
-  const editing = equalPath(state.cursor, oldPath)
+  const editing = subsetThoughts(state.cursor, oldPath)
```

Comment:
Let's rename this to `isEditingPath` or `isPathInCursor`. The variable name should change to match the new semantics.

Reply:
changed to isPathInCursor